app/src/main/java/app/revanced/integrations/patches/spoof/requests/StoryBoardRendererRoutes.java

```diff
@@ -6,21 +6,20 @@
 import java.io.IOException;
 import java.net.HttpURLConnection;
 
-/** @noinspection unused*/
-public final class StoryBoardRendererRoutes {
+final class StoryBoardRendererRoutes {
     private static final String YT_API_URL = "https://www.youtube.com/youtubei/v1/";
-    static final Route GET_STORYBOARD_SPEC = new Route(
+    private static final Route.CompiledRoute GET_STORYBOARD_SPEC = new Route(
             Route.Method.POST,
             "player" +
                     "?fields=storyboards.playerStoryboardSpecRenderer," +
                     "storyboards.playerLiveStoryboardSpecRenderer"
-    );
+    ).compile();
 
     private StoryBoardRendererRoutes() {
     }
 
-    public static HttpURLConnection getPlayerResponseConnectionFromRoute() throws IOException {
-        var connection = Requester.getConnectionFromRoute(YT_API_URL, GET_STORYBOARD_SPEC);
+    static HttpURLConnection getPlayerResponseConnectionFromRoute() throws IOException {
```

Comment:
Just like https://github.com/revanced/revanced-integrations/blob/693ef08c6c2ebb2a4dde9194583ac77ae3af9fc6/app/src/main/java/app/revanced/integrations/returnyoutubedislike/requests/ReturnYouTubeDislikeRoutes.java#L24 should not a parameter be used for consistency reasons?

Reply:
This would also apply in the case of https://[github.com/revanced/revanced-integrations/blob/ae18edd047d7979307bc28f28db17bae2c5cc226/app/src/main/java/app/revanced/integrations/returnyoutubedislike/requests/ReturnYouTubeDislikeApi.java#L406](https://github.com/revanced/revanced-integrations/blob/ae18edd047d7979307bc28f28db17bae2c5cc226/app/src/main/java/app/revanced/integrations/returnyoutubedislike/requests/ReturnYouTubeDislikeApi.java#L406)

Reply:
Alright, I'll look into it.

Reply:
You added a parameter but the signature of the getConnection method still does not match that of the other Route classes. The other route classes compile the route additionally but our class does not need any current route to be compiled. This is why our class uses the `getConnectionFromCompiledRoute` method instead of the other classes using getConnectionFromRoute`. This causes inconsistency between all the "Routes" classes. One way to solve this would be that the class `CompiledRoute` could inherit from `Route` and similar OOP tricks. I have pushed one more consistency fix that can be done for now. Instead of an unused params parameter, the compiled route is now passed as a parameter.